src/tensors/tensoroperations.jl

```diff
@@ -144,6 +144,29 @@ TO.tensorcost(t::AbstractTensorMap, i::Int) = dim(space(t, i))
 # IMPLEMENTATONS
 #----------------
 
+# Backend implementation
+# ----------------------
+# TODO: figure out a name
+# TODO: what should be the default scheduler?
+# TODO: should we allow a separate scheduler for "blocks" and "subblocks"
```

Comment:
Yes please. Or we should at least have different defaults. But given that a method like `tensorcontract!` wants to do both a permutation step (subblocks) and a multiplication (blocks), it would need to know about both schedulers.